### PR TITLE
Impreza 2017: add missing eps FW

### PR DESCRIPTION
**Pull request by chengyu-k**

### CODE REVIEW

**Comment by vanillagorillaa** [comment]
Be sure to include the dongleID of the device that these firmware versions were gathered from

**Comment by chengyu-k** [comment]
Here is my dongle ID for my Comma three: d61c6b311ca10e7d
Thank you for your helping.

File: selfdrive/car/subaru/values.py [hunk 1]
```diff
@@ -126,6 +127,8 @@ class SubaruCarInfo(CarInfo):
       b'\x00\x00c\xf4\x00\x00\x00\x00',
       b'\x00\x00d\xdc\x00\x00\x00\x00',
       b'\x00\x00dd\x00\x00\x00\x00',
+      b'\x00\x00d)\x1f@ \x07',
+      b'\xf1\x00\xac\x02\x00',
```
Comment (sshane) [inline selfdrive/car/subaru/values.py:131]:
```suggestion
```

Comment (sshane) [reply selfdrive/car/subaru/values.py:131]: I don't see this in any recent routes

**Pull request merged**